presto-docker/bin/entrypoint

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash -xeu
+
+# Copy config files, if they exist, to /usr/lib/presto/etc. This is necessary
+# for Kubernetes as Secrets and ConfigMaps are mounted read only.
+if [[ -d /usr/lib/presto/conf ]]; then
+    if [[ -d /usr/lib/presto/conf/catalog ]]; then
```

Comment:
In Kubernetes it's quite possible that the node configuration and catalog config will be mounted separately. Here I want to delete the appropriate destination files and then copy the node and catalog config if it's present, otherwise I want to only copy the node config.

Reply:
Actually on further review, I've eliminated the differentiation for the existence of catalog configs.